src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -0,0 +1,2552 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// #define USING_HASH_SET
```

Comment:
Can the #ifs for USING_HASH_SET be deleted, or should this be added to the .csproj?